.github/workflows/build.yaml

```diff
@@ -22,6 +21,11 @@ jobs:
     with:
       script: "ci/build_python.sh"
     secrets: inherit
+  upload-conda:
+    needs:
+      - conda-python-build
+    uses: ./.github/workflows/conda-upload-packages.yaml
+    secrets: inherit
```

Comment:
Adding this here, with these triggers:

https://github.com/rapidsai/legate-boost/blob/166c98d3c1d59219ca43aab72c162e5ac9beefa9/.github/workflows/build.yaml#L7-L17

Means that new `legate-boost` packages will automatically be published to the `legate` channel in the following situations.

**Scenario 1: New packages pushed to the `experimental` label, with version like `{YY}.{MM}.{patch}.dev{n}`**

* new commit pushed to `main`
* new commit pushed to any branch named `release/v{YY}.{MM}.{patch}`
* new tag pushed which does not exactly match the pattern `v[0-9][0-9].[0-9][0-9].[0-9][0-9]`
* someone manually clicked the "run workflow" button in GitHub Actions UI

**Scenario 2: New packages pushed to the `main` label, with version like `{YY}.{MM}.{patch}` (i.e. a stable release)**

* new tag pushed exactly matching the pattern `v[0-9][0-9].[0-9][0-9].[0-9][0-9]`